### DISCUSSION TITLE
Default system time

### DISCUSSION THREAD

**Discussion by Rubisco16** [inventree/InvenTree#2814] [Q&A] [answered]
Hi there,
Can I change the default system time zone of my inventree system? When I export the stock item tracking imformation, the default time zone is GMT. If I want to change to other time zone, what can I do?

**Comment by SchrodingersGat** [comment, marked as answer]
Change the "timezone" option in `config.yaml` file 

https://inventree.readthedocs.io/en/latest/start/config/ (Ctrl+F "timezone")

**Reply by Rubisco16** [reply]
environment:
- INVENTREE_TIMEZONE=Asia/Shanghai

So I changed the environment variable like this, but it still didn't work. Is there something wrong with this sentence?

**Reply by SchrodingersGat** [reply]
What isn't working, precisely? What did you expect to change?

**Reply by Rubisco16** [reply]
In admin page, "stock item tracking", the date time will not change.

**Comment by Rubisco16** [comment]
Thank you!!!
